src/firejail/main.c

```diff
@@ -2187,34 +2187,21 @@ int main(int argc, char **argv, char **envp) {
 		else if (strncmp(argv[i], "--name=", 7) == 0) {
 			cfg.name = argv[i] + 7;
 			if (strlen(cfg.name) == 0) {
-				fprintf(stderr, "Error: please provide a name for sandbox\n");
+				fprintf(stderr, "Error: invalid sandbox name: cannot be empty\n");
```

Comment:
Nitpick. The double `:` reads a bit awkward. What do you think of "Error: invalid sandbox name (cannot be empty)\n"?